definitions/procedures/foreman_tasks/delete/old.rb

```diff
@@ -0,0 +1,32 @@
+module Procedures::ForemanTasks
+  module Delete
+    class Old < ForemanMaintain::Procedure
```

Comment:
It seems we could still unify the `delete/old.rb` and `delete/pending_state.rb` just by having a `delete.rb`, that would take the properties of what task to delete, based on arguments